pkg/sdk/identifier_parsers_test.go

```diff
@@ -250,3 +271,31 @@ func Test_ParseObjectIdentifierString(t *testing.T) {
 		})
 	}
 }
+
+func Test_ParseFunctionArgumentsFromString(t *testing.T) {
+	testCases := []struct {
+		Arguments string
+		Expected  []DataType
+		Error     string
+	}{
+		{Arguments: `()`, Expected: []DataType{}},
+		{Arguments: `(FLOAT, NUMBER, TIME)`, Expected: []DataType{DataTypeFloat, DataTypeNumber, DataTypeTime}},
```

Comment:
please add a case when e.g. VARCHAR(200) is used

Reply:
Added test cases (also added cases like `Number(10, 2)` and explained it's not supported for now as it's not needed for any use case)